examples/jss/src/components/footer/footer.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component, PropTypes } from 'react';
+import useSheet from 'react-jss';
+
+import ReplyIcon from '../../../../shared/assets/reply.svg';
+import RetweetIcon from '../../../../shared/assets/retweet.svg';
+import LikeIcon from '../../../../shared/assets/like.svg';
+import MoreIcon from '../../../../shared/assets/more.svg';
+
+const styles = {
+  '@keyframes liked': {
+    '50%': {
+      transform: 'scale(1.2)',
+    },
+    '100%': {
+      tranform: 'scale(1)',
+    },
+  },
+  date: {
+    paddingBottom: '.98438rem',
+    color: '#8899a6',
+  },
+  counters: {
+    borderTop: '1px solid #e1e8ed',
+    padding: '.98438rem 0',
+    textTransform: 'uppercase',
+  },
+  value: {
+    fontWeight: '700',
+  },
+  label: {
+    color: '#8899a6',
+    fontSize: '.85rem',
+  },
+  favorite: {
+    display: 'inline-block',
+    marginLeft: '1.96875rem',
+  },
+  actions: {
+    alignItems: 'center',
+    borderBottom: '1px solid #e1e8ed',
+    borderTop: '1px solid #e1e8ed',
+    color: '#8899a6',
+    display: 'flex',
+    fontSize: '1.5rem',
+    height: '3.28125rem',
+    width: '100%',
+  },
+  icon: {
+    display: 'flex',
+    flexGrow: '1',
+    justifyContent: 'center',
+    textAlign: 'center',
+  },
+  button: {
+    display: 'flex',
+    flexGrow: '1',
+    justifyContent: 'center',
+    textAlign: 'center',
+    background: 'none',
+    border: 'none',
+    color: 'inherit',
+    cursor: 'pointer',
+    fontSize: 'inherit',
+    outline: 'none',
+  },
+  liked: {
+    animationName: 'liked',
+    animationDuration: '.25s',
+    color: '#e81c4f',
+  },
+};
+
+class Footer extends Component {
```

Comment:
Not a fan of decorators since they aren't standardized yet and rely on a legacy babel transform. 😁 See this dicussion: https://github.com/facebookincubator/create-react-app/issues/107

Specifically this part:

> Decorators are currently at "stage 1" in the ecmascript-standardization process, and it isn't until "stage 3" that all the syntax and semantics are complete, so I think it makes sense to wait.